addon/components/property-group.hbs

```diff
@@ -20,40 +20,48 @@
       </div>
     {{/if}}
     {{!-- NOTE: Children can be property-groups as well as fields --}}
-    {{#each this.children as |child index|}}
-      {{#if (not (is-property-group child.displayType))}}
-        <div class="au-u-margin-bottom-small">
-          {{component (component-for-display-type child.displayType show=@show)
-                      field=child
-                      form=@form
-                      formStore=@formStore
-                      graphs=@graphs
-                      sourceNode=@sourceNode
-                      forceShowErrors=@forceShowErrors
-                      cacheConditionals=@cacheConditionals
-                      show=@show
-          }}
-          {{! template-lint-disable no-triple-curlies}}
-          <AuHelpText>{{{child.help}}}</AuHelpText>
-        </div>
-      {{else}}
-        <PropertyGroup
-                @level={{this.nextLevel}}
+    {{#each this.children as |field index|}}
+      {{#if field.displayType}}
+        {{#if (not (is-property-group field.displayType))}}
```

Comment:
Wouldnt it be nicer to remove `(not` and just switch the conditions around?

```
  {{#if (is-property-group field.displayType)}}
     <PropertyGroup
        @level={{this.nextLevel}}
        @form={{@form}}
        @group={{field}}
        @formStore={{@formStore}}
        @graphs={{@graphs}}
        @sourceNode={{@sourceNode}}
        @forceShowErrors={{@forceShowErrors}}
        @cacheConditionals={{@cacheConditionals}}
        @show={{@show}}
      />
  
      {{#if @last}}
        {{#if (not-eq index (sub this.children.length 1))}}
          <AuHr @size="large"/>
        {{/if}}
      {{else}}
        <AuHr @size="large"/>
      {{/if }}

  {{else}}
    <div class="au-u-margin-bottom-small">
      {{#let (ensure-safe-component
        (component-for-display-type field.displayType show=@show))
        as |FormField|
      }}
        <FormField
          @field={{field}}
          @form={{@form}}
          @formStore={{@formStore}}
          @graphs={{@graphs}}
          @sourceNode={{@sourceNode}}
          @forceShowErrors={{@forceShowErrors}}
          @cacheConditionals={{@cacheConditionals}}
          @show={{@show}}
        />
      {{/let}}

      {{! template-lint-disable no-triple-curlies}}
      <AuHelpText>{{{field.help}}}</AuHelpText>
    </div>
  {{/if}}
        
```

Reply:
Yes, I just adjusted what was already there. I'll swap them around :+1:

Reply:
Should be good now.

Reply:
Thanks! :smile: